knowledge/shaders.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+title: Shaders
+parent: Knowledge
+---
+
+# Shaders
+
```

Comment:
Maybe include a section about how shaders can only be used with the RTX Beta? Just so people don't get confused?

Reply:
But you can use shaders on any version of Minecraft. I also verified, that those work on android version of Minecraft.